pkgs/development/python-modules/add-trailing-comma/default.nix

```diff
@@ -8,7 +8,7 @@
 
 buildPythonPackage rec {
   pname = "add-trailing-comma";
-  version = "2.2.3";
+  version = "2.3.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

Comment:
```suggestion
```
this (and the `pythonOlder` package) can be removed due to https://github.com/asottile/add-trailing-comma/compare/v2.2.3...v2.3.0

Reply:
Upstream still have the [restriction](https://github.com/asottile/add-trailing-comma/blob/main/setup.cfg#L23) on the Python version.

Reply:
Ah, I just looked at the removed classifiers, you're right